src/main/resources/assets/css/main.less

```diff
@@ -123,6 +123,36 @@ a,
   background-color: #323539;
```

Comment:
`.modal-body-no-top` is not used anywhere anymore. Please make sure to clean up after renaming, or using other styles/removing styles